src/useFirebaseUser.js

```diff
@@ -75,6 +79,7 @@ const useFirebaseUser = () => {
     setUser(firebaseUser)
     setInitialized(true)
     await setAuthCookie(firebaseUser)
+    setIsAuthCookieRequestComplete(true)
```

Comment:
Problem: this is persisted across client navigation, so following pages will not wait for the request. We need to reset this to false.

Reply:
Also need to cancel the request on unmount.